grossmont/computer-science-and-cybersecurity-organization/index.html

```diff
@@ -29,6 +29,6 @@
 </div>
   <script src="/jquery.min.js"></script>
   <!-- <script src="/grossmont-cybersecurity-club/sticky-header.js"></script> -->
+<script src="/theme.js"></script>
```

Comment:
The theme.js file does not need to be linked index.html because there is actually no element with the id of icon in this html file, only the header, so you only need to link it in header.html, which would be enough